includes/validation/class-amp-validation-manager.php

```diff
@@ -99,13 +99,6 @@ class AMP_Validation_Manager {
 	 */
 	const VALIDATION_ERROR_TERM_STATUS_QUERY_VAR = 'amp_validation_error_term_status';
 
-	/**
```

Comment:
I searched for the deleted code in this file on wpdirectory.net and didn't find any other plugins calling it. So safe to delete.